scripts/run_forge.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+
+# Copyright (c) Aptos
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# This script runs Forge using the current configured kubectl context
+# It is designed to be invoked within Github Actions, but can be run locally
+# if the necessary environment variables are set.
+#
+
+# for calculating regression
+TPS_THRESHOLD=1500
```

Comment:
What were the threshold for TPS and latency we were using earlier?

Reply:
I was under impression that multiple tests can't use the same machines with the new setup. Can you clarify?

Reply:
There's two components: (1) the test runner that drives the tests and submits txns, and (2) the network of validators/fullnodes/haproxy e.g. the Deployment.

The test runner can be run locally from your laptop or you can schedule a pod on k8s (which is what we're doing in CI). The validators/fullnodes are guaranteed their own machines due to their resource allocation, but there's nothing preventing test runners from running on the same pod

Reply:
The test runner itself should not consume too much resources. Do we ensure each test runner is ran inside its own k8 container though?